impl/sql/mutationstorage/queue.go

```diff
@@ -58,40 +58,40 @@ func (m *Mutations) Send(ctx context.Context, domainID string, update *pb.EntryU
 	}
 	// TODO(gbelvin): Implement retry with backoff for retryable errors if
 	// we get timestamp contention.
-	return m.send(ctx, domainID, shardID, mData, time.Now())
+	return m.send(ctx, domainID, logID, mData, time.Now())
 }
 
-// randShard returns a random, enabled shard for domainID.
-func (m *Mutations) randShard(ctx context.Context, domainID string) (int64, error) {
+// randLog returns a random, enabled log for domainID.
+func (m *Mutations) randLog(ctx context.Context, domainID string) (int64, error) {
 	// TODO(gbelvin): Cache these results.
-	var shardIDs []int64
+	var logIDs []int64
 	rows, err := m.db.QueryContext(ctx,
-		`SELECT ShardID from Shards WHERE DomainID = ? AND Enabled = ?;`,
+		`SELECT LogID from Logs WHERE DomainID = ? AND Enabled = ?;`,
```

Comment:
Since `Enabled` is always `true`, could you pop it into the SQL itself rather than as a const parameter?

Reply:
I could, but I was worried about `True` vs `true` vs `TRUE` for the specific SQL dialect.

Reply:
are there SQL dialects where these are case sensitive?